soccer/gameplay/skills/angle_receive.py

```diff
@@ -135,6 +136,17 @@ def execute_running(self):
             main.system_state().draw_line(self._kick_line,
                                           constants.Colors.Red, "Shot")
 
+    # This depends on the superclasse's implementation somewhat
+    # If we stop using capture in pass_receive, this needs to be modified.
+    def on_enter_receiving(self):
```

Comment:
Pass Receive needs a fresh object every time it runs (this would break multiple coordinated passes, as in testCoordinatedPass). I could pass in a lambda that generates a Capture object. That probably would be a lot cleaner.